File: 프로그래머스/unrated/181912. 배열 만들기 5/배열 만들기 5.py
def solution(intStrs, k, s, l):
    answer = []
    
    for intStr in intStrs:
        substring = intStr[s:s+l]
        integer_value = int(substring)
        
        if integer_value > k:
            answer.append(integer_value)
            
    return answer

File: 프로그래머스/0/181912. 배열 만들기 5/배열 만들기 5.java
import java.util.*;
class Solution {
    public int[] solution(String[] intStrs, int k, int s, int l) {
        List<Integer> result = new ArrayList<>();
        
        for(String str:intStrs){
            int a = Integer.parseInt(str.substring(s,s+l));
            if(a>k){
                result.add(a);
            }
        }
        return result.stream().mapToInt(i->i).toArray();
    }
}